include/sensors/core/mmr920.hpp

```diff
@@ -26,12 +26,16 @@
  * Leaving them alone for now.
  */
 namespace sensors {
-namespace mmr920C04 {
+namespace mmr920 {
 constexpr uint16_t ADDRESS = 0x67 << 1;
 
-// Pressure cannot be measured beyond +/-8226.4F
+// Pressure cannot be measured beyond +/-8226.4F on the old sensors
+// New sensors have double the threshold but half the fidelity
+#if PCBA_PRIMARY_REVISION == 'e'
```

Comment:
Regardless of whether we keep the revision handling or some other method, we should move this ifdef to somewhere closer to an application root, and have this file parametrized on what kind of sensor it should represent.